app/Models/GamesDeveloper.php

```diff
@@ -38,11 +38,11 @@ class GamesDeveloper extends Model
 
     protected $guarded = [];
 
-    public function developer(){
+    public function developer() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->